docker/supabase/volumes/db/init/05-github-schema.sql

```diff
@@ -0,0 +1,29 @@
+CREATE TABLE IF NOT EXISTS github_users (
+  id serial PRIMARY KEY,
+  username text, -- the user name of the user, known as the `login` field in GitHub API responses
+  user_type text check ( user_type IN ('User', 'Organization') )
```

Comment:
Why use Title Case vs. `user` and `organization`?

Reply:
Because in the GitHub API the use title case. I can change it to lowercase, but we need to make sure that we always make the types lowercase.

Reply:
Interesting.  If they do it, maybe we should too?  Just seems odd to me.

Reply:
It is weird, I agree :/
I think we can always make it lowercase with the datatype that @DukeManh used for the emails in the `telescope`-specific schema